test/extended/openstack/cloud-provider-config.go

```diff
@@ -104,6 +104,70 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The Openshift", f
 				}
 			}
 		})
+
+		//https://bugzilla.redhat.com/show_bug.cgi?id=2074471
+		g.It("should update use-octavia property depending on the NetworkType", func() {
+
+			userConfigMap := config_params{
+				namespace: "openshift-config",
+				name:      "cloud-provider-config",
+				key:       "config",
+				skip:      nil,
+			}
+
+			ccmConfigMap := config_params{
+				namespace: "openshift-cloud-controller-manager",
+				name:      "cloud-conf",
+				key:       "cloud.conf",
+				skip:      nil,
+			}
+
+			section_name := "LoadBalancer"
+			property_name := "use-octavia"
+
+			cfg, err := e2e.LoadConfig()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			c, err := configv1client.NewForConfig(cfg)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			network, err := c.ConfigV1().Networks().Get(context.Background(), "cluster", metav1.GetOptions{})
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			user_cfg, err := getConfig(clientSet,
+				userConfigMap.namespace,
+				userConfigMap.name,
+				userConfigMap.key)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			ccm_cfg, err := getConfig(clientSet,
+				ccmConfigMap.namespace,
+				ccmConfigMap.name,
+				ccmConfigMap.key)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			user_property_value, err := getPropertyValue(section_name, property_name, user_cfg)
+			o.Expect(err).NotTo(o.HaveOccurred(), "Error getting the property %q in section %q on configMap "+
+				"%q (namespace: %q)", property_name, section_name, userConfigMap.name, userConfigMap.namespace)
```

Comment:
We could probably put this into the function itself, so that we don't repeat it in line 152?

Reply:
I prefer not to through errors on functions and handle them on the test itself. I believe this is better for debugging failing testcases.

Reply:
Okay.